webcompat/form.py

```diff
@@ -418,6 +432,8 @@ def build_formdata(form_object):
     domain = domain_name(normalized_url)
     problem_summary = get_problem_summary(form_object.get('problem_category'))
 
+    switch_problem_subtype(form_object.get('problem_category'))
```

Comment:
This is a function which doesn't assign anything. What was the intent?

Reply:
You are right. I created this function to extract the problem sub-type. Then I integrated all the additional fields into existing fields (sub-type went to description), and there is no need for this function any more. I will eliminate the code where this function is defined, and the code where it is called.